java/values.yaml

```diff
@@ -21,3 +21,13 @@ useInterpodAntiAffinity: false
 pdb:
   enabled: true
   maxUnavailable: 50%
+
+## Postgres chart configuration
+## https://github.com/helm/charts/blob/master/stable/postgresql/values.yaml
+
+postgresql:
+  ## Whether to deploy the Postgres Chart or not
+  enabled: false
+  ## Disable Postgres Persistence
+  persistence:
+    enabled: false
```

Comment:
Probably it would be convenient to have default configurations here for host, user, password, port and ssl.

Reply:
Added default username&password&database